### PR TITLE
Scan

### PR DESCRIPTION
**Pull request by DennisTarik**

### CODE REVIEW

File: src/app/utils/usePostDocument.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { useState } from 'react';
+
+function usePostDocument() {
```
Comment (lmachens) [inline src/app/utils/usePostDocument.ts:3]: I like this, but `usePostDocument` is not used in your project. Maybe you like to use it in `AddDocumentForm`?

**Review by lmachens — COMMENTED** [review]
Good job! Could you create a description of what you like to do and what this pull request is about?

File: src/app/components/Scan/Scan.tsx [hunk 2]
```diff
@@ -29,20 +28,18 @@ function Scan(): JSX.Element {
...
-      {recognizedProgress ? (
-        <Progress
-          progress={recognizedProgress.progress * 100}
-          status={recognizedProgress.status}
-        />
+      {!text && progress && (
+        <Progress progress={progress.progress} status={progress.status} />
+      )}
+      {text ? (
+        <div>{text && <AddDocumentForm text={text} />}</div>
```
Comment (AnkePfei2fer) [inline src/app/components/Scan/Scan.tsx:35]: To make your code more clearly, you could transfer these 2 conditions also to your content variable.

**Review by AnkePfei2fer — COMMENTED** [review]
lgtm 🤖

**Review by riittagerhards — APPROVED** [review]
lgtm 👍  In next step you could add styling and add a custom hook for the fetch.

**Review by manusmd — APPROVED** [review]
lgtm 🔥

File: src/app/utils/usePostDocument.ts (hunk 1 above)
Comment (DennisTarik) [reply src/app/utils/usePostDocument.ts:3]: Mhh ?

**Pull request merged**